grapher/sourcesTab/SourcesTab.tsx

```diff
@@ -47,7 +49,7 @@ export class SourcesTab extends React.Component<{
         return (
             <div key={slug} className="datasource-wrapper">
                 <h2>
-                    {column.name}{" "}
+                    {decodeHTML(column.name)}{" "}
```

Comment:
This is a database issue on our side – we shouldn't be storing HTML entities in variable names, but unicode symbols. This is an old import (from 2018) that wasn't properly cleaned.

I think it's a good idea to drop it from the PR and we can fix it next time we run the importer.

Reply:
Yes, that makes more sense. I've reverted this in [c49acfc](https://github.com/owid/owid-grapher/pull/1202/commits/c49acfc69ce9ddbf0195bdb9bc54ede83b300afc).